dask_sql/physical/rel/logical/aggregate.py

```diff
@@ -61,6 +61,21 @@ def __init__(self, numerical_aggregation, non_numerical_aggregation=None):
             non_numerical_aggregation or numerical_aggregation
         )
 
+    def get_dtype_relevant_aggregation(self, dtype):
+        numeric_agg = self.numerical_aggregation
+
+        if pd.api.types.is_numeric_dtype(dtype):
+            return numeric_agg
+
+        # Todo: Add Categorical when support comes to dask-sql
+        if numeric_agg in ["min", "max", "first"]:
```

Comment:
Since `first` is the same aggregation for both numeric and non-numeric, no special check is needed for this aggregation. Though for clarity it might make sense to include it here, as well as include `count` which falls in the same criteria.

Reply:
Resolved with removing first  (see commit: https://github.com/dask-contrib/dask-sql/pull/236/commits/89822d92dd9436262d4096fcbecbb371207c6bfe) .

Thanks for the review , good catch on this.